v1/transform-global.yaml

```diff
@@ -0,0 +1,227 @@
+info:
+  version: '1.0.0'
+  title: Global Transform API
+  description: Global Transform API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /html/{from}/to/{to}{/provider}:
```

Comment:
Good point @gwicke . To be consistent with the other transform end points, how about `transform/html/from/{from_lang}/to/{to_lang}{/provider}` ?